Amethyst/Preferences/UserConfiguration.swift

```diff
@@ -533,13 +533,11 @@ class UserConfiguration: NSObject {
     func runningApplicationFloatingBundle(_ runningApplication: BundleIdentifiable) -> FloatingBundle? {
         let floatingBundles = self.floatingBundles()
 
+        let bundleIdentifier = runningApplication.bundleIdentifier ?? "*"
```

Comment:
Maybe this should just be the empty string? `.*` should still match that.

Reply:
To be honest I am so new to this project that I don't really have opinion about that :) If you prefer it to be empty string, I can test it if it works and change accordingly?

Reply:
I think conceptually I would prefer an empty string.

Reply:
Done.